React/Base/RCTConvert.m

```diff
@@ -1030,6 +1050,13 @@ + (RCTUIColor *)UIColor:(id)json // TODO(OSS Candidate ISS#2710739)
         RCTLogConvertError(json, @"a UIColor. Expected a dynamic appearance aware color.");
         return nil;
       }
+// [TODO(macOS GH#750)
```

Comment:
should this have ifdef of target_os_osx? systemeffect color doesn't make sense in ios

Reply:
Yes, good catch.